packages/widget_toolkit/test/lib_search_picker/blocs/search_picker_test.dart

```diff
@@ -31,7 +31,6 @@ void main() {
         act: (bloc) async {},
         state: (bloc) => bloc.states.itemsList,
         expect: [
-          Result.loading(),
```

Comment:
Why the loading is removed here ? Don't we need to show anything to the user while the data is loading ?

Reply:
@StanevPrime ^